### PR TITLE
Feature/#112 sign in

### PR DESCRIPTION
**Pull request by varathar53**

### CODE REVIEW

File: app/src/main/java/ch/epfl/sdp/game/DatabaseHelper.java [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+            String sqlQuery = String.format("INSERT INTO %s (%s, %s) VALUES (?, ?)",
+                    USERS_TABLE_NAME, USERS_EMAIL, USERS_PASSWORD, email, password);
+            SQLiteStatement stmt = writableDatabase.compileStatement(sqlQuery);
+            stmt.bindString(1, email);
+            stmt.bindString(2, password);
+            stmt.executeInsert();
+        }
+        catch (Exception ex)
+        {
+            int a = 5;
```
Comment (kangpl) [inline app/src/main/java/ch/epfl/sdp/game/DatabaseHelper.java]: What's the purpose of this line of code, I mean int a = 5. Shall we, for example, print the exception message?

File: app/src/main/java/ch/epfl/sdp/game/DatabaseHelper.java [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    @Override
+    public void onCreate(SQLiteDatabase sqLiteDatabase)
+    {
+        try
+        {
+            sqLiteDatabase.execSQL(USERS_CREATE_TABLE);
+        }
+        catch (Exception ex)
+        {
+            int a = 5;
```
Comment (kangpl) [inline app/src/main/java/ch/epfl/sdp/game/DatabaseHelper.java]: Same as before

File: app/src/main/java/ch/epfl/sdp/game/DatabaseHelper.java [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+    public void deleteAllUsers() {
+        try
+        {
+            SQLiteDatabase writableDatabase = getWritableDatabase();
+            writableDatabase.execSQL("DELETE FROM " + USERS_TABLE_NAME);
+        }
+        catch (Exception ex)
+        {
+            int a = 5;
```
Comment (kangpl) [inline app/src/main/java/ch/epfl/sdp/game/DatabaseHelper.java]: Same as before

File: app/src/main/java/ch/epfl/sdp/game/DatabaseHelper.java (hunk 1 above)
Comment (varathar53) [reply app/src/main/java/ch/epfl/sdp/game/DatabaseHelper.java]: Yeah thats a code Idea!

Comment (varathar53) [reply app/src/main/java/ch/epfl/sdp/game/DatabaseHelper.java]: changed

File: app/src/main/java/ch/epfl/sdp/game/DatabaseHelper.java (hunk 2 above)
Comment (varathar53) [reply app/src/main/java/ch/epfl/sdp/game/DatabaseHelper.java]: changed

File: app/src/main/java/ch/epfl/sdp/game/DatabaseHelper.java (hunk 3 above)
Comment (varathar53) [reply app/src/main/java/ch/epfl/sdp/game/DatabaseHelper.java]: changed

**Pull request merged**